<?xml version="1.0" encoding="UTF-8"?><testrun name="TestMatrixPublic" project="AI" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testsuite name="tests.TestMatrixPublic" time="0.767" displayname="TestMatrixPublic" uniqueid="[engine:junit-vintage]/[runner:tests.TestMatrixPublic]">
    <testcase name="testa6" classname="tests.TestMatrixPublic" time="0.767" displayname="testa6" uniqueid="[engine:junit-vintage]/[runner:tests.TestMatrixPublic]/[test:testa6(tests.TestMatrixPublic)]">
      <error>java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0&#13;
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)&#13;
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)&#13;
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)&#13;
	at java.base/java.util.Objects.checkIndex(Objects.java:359)&#13;
	at java.base/java.util.ArrayList.get(ArrayList.java:427)&#13;
	at code.GeneralSearch.generalSearch(GeneralSearch.java:128)&#13;
	at code.Matrix.solve(Matrix.java:302)&#13;
	at tests.TestMatrixPublic.testa6(TestMatrixPublic.java:108)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)&#13;
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:288)&#13;
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:282)&#13;
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)&#13;
	at java.base/java.lang.Thread.run(Thread.java:831)&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
